test/client/scraper/test_error_metadata.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+import json
+from client.query_utils import QueryParams
+from client.constants import ResponseType
+import client.api_handler
+import client.api
+
+
+class TestErrorMetadata(unittest.TestCase):
+
+    test_url = "http://samiya.ca/about.html"
+
+    def test_error(self):
```

Comment:
in case you actually do add in an about.html page one day, I would do a url that's more likely to be invalid:
eg. https://github.com/invalidurl